lib/nlp/nlp-manager.js

```diff
@@ -442,11 +442,54 @@ class NlpManager {
   }
 
   /**
-   * Save the NLP manager information into a file.
-   * @param {String} srcFileName Filename for saving the NLP manager.
+   * Load NLP manager information from a string.
+   * @param {String} data JSON string to load NLP manager information from.
    */
-  save(srcFileName) {
-    const fileName = srcFileName || 'model.nlp';
+  import(data) {
+    const clone = JSON.parse(data);
+
+    this.settings = clone.settings;
+    this.languages = clone.languages;
+    this.nerManager.load(clone.nerManager);
+    this.intentEntities = clone.intentEntities;
+    this.intentDomains = clone.intentDomains || {};
+    this.nlgManager.responses = clone.responses;
+    for (let i = 0, l = clone.classifiers.length; i < l; i += 1) {
+      const classifierClone = clone.classifiers[i];
+      this.addLanguage(classifierClone.language);
+      const classifier = this.classifiers[classifierClone.language];
+
+      classifier.docs = classifierClone.docs;
+      classifier.features = classifierClone.features;
+      const lrc = classifier.settings.classifier;
+      const { logistic } = classifierClone;
+
+      lrc.observations = {};
+      Object.entries(logistic.observations).forEach(([label, matrix]) => {
+        lrc.observations[label] = [];
+        matrix.forEach((row, rowIndex) => {
+          // Create a array filled with as many zeros as features
+          const features = Array(classifier.features.length).fill(0);
+          // Set the features for the positions stored with 1. The others remains as zero.
+          row.forEach((featPosition) => {
+            features[featPosition] = 1;
+          });
+          lrc.observations[label][rowIndex] = features;
+        });
+      });
+
+      lrc.labels = logistic.labels;
+      lrc.classifications = logistic.classifications;
+      lrc.observationCount = logistic.observationCount;
+      lrc.theta = logistic.theta;
+    }
+  }
+
+  /**
+   * Export NLP manager information as a string.
+   * @returns {String} NLP manager information as a JSON string.
```

Comment:
Explain param _minified_, so the developer  knows the usage reading the doc.

Reply:
Oops! Missed that one. Will fix now.

Reply:
Just pushed these changes.